source/jormungandr/tests/check_utils.py

```diff
@@ -1376,3 +1376,10 @@ def get_schedule(scs, sp_uri, line_code):
             if rp_sched['stop_point']['id'] == sp_uri and rp_sched['route']['line']['code'] == line_code
         )
     ]
+
+
+def get_disruption(disruptions, disrupt_id):
+    for d in disruptions:
+        if d['id'] == disrupt_id:
+            return d
+    return None
```

Comment:
In a more pythonic way, you could also use [list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) here :
````python
disruption = next((d for d in disruption if d['id']==disrupt_id), None)